### PR TITLE
#2325 Make connectors table rows clickable and #2076 Implement connec…

### LINKED ISSUE #2076: Implement connectors sorting
1. Migrate table to new table component
2. Implement sorting
3. Related issue https://github.com/provectus/kafka-ui/issues/2325
The list is unsorted at all

image

### LINKED ISSUE #2325: Make connectors table rows clickable

### PR DESCRIPTION
**Pull request by Kris-K-Dev**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
Fixes  #2076 and #2325
Make connectors table rows clickable and Implement connectors sorting

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [x] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**
Screenshot 2022-09-30 at 21 19 25

### CODE REVIEW

**Comment by workshur** [comment]
@Haarolean pls take a look 👀

**Review by Haarolean — CHANGES_REQUESTED** [review]
status sorting doesn't seem to work proeprly
image

**Comment by Kris-K-Dev** [comment]
> status sorting doesn't seem to work proeprly image

Could you please check it now? Thank you

**Comment by Haarolean** [comment]
@VladSenyuta it seems like we broke something here

**Review by Haarolean — APPROVED** [review]

**Comment by VladSenyuta** [comment]
> @VladSenyuta it seems like we broke something here

fixed

**Review by VladSenyuta — APPROVED** [review]

**Comment by Haarolean** [comment]
@Kris-K-Dev good to merge
@workshur we need an approve here

**Review by workshur — COMMENTED** [review]
Pls increase test coverage. It should be at least 80%

File: kafka-ui-react-app/src/components/Connect/List/TopicsCell.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  row,
+}) => {
+  const { topics } = row.original;
+
+  const { clusterName } = useAppParams<ClusterNameRoute>();
+  return (
+    <S.TagsWrapper>
+      {topics?.map((t) => (
+        <Tag key={t} color="gray">
+          <Link color="#4C4CFF" to={clusterTopicPath(clusterName, t)}>
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Connect/List/TopicsCell.tsx:21]: Pls use styled components attr to change link color. Or update Tag styles to show link correctly

**Review by workshur — APPROVED** [review]

**Pull request merged**